config.json

```diff
@@ -721,6 +721,14 @@
           "strings"
         ]
       },
+      {
+        "slug": "yacht",
+        "name": "Yacht",
+        "uuid": "32341692-02df-48a0-b696-1cfe93dc6862",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2
```

Comment:
Maybe make this a `3` difficulty?

Reply:
Right. What does Julia do?

Reply:
Apparently not that so I can bump the difficulty.

Reply:
Done.